packages/template-retail-react-app/app/commerce-api/einstein.js

```diff
@@ -67,11 +67,19 @@ class EinsteinAPI {
                 altId: '',
                 altIdType: ''
             }
+        } else if (product.type && product.type.set) {
+            // handle sets for viewProduct
+            return {
+                id: product.id,
+                sku: product.id,
```

Comment:
I think for product sets we can avoid having a separate case and let any product that doesn't have variants fall through to the else block. 

That's because I'm not sure if it's relevant for us to be sending 'sku' if it is the same as the regular 'id'.

Otherwise, the changes look good to me!

Reply:
The einstein test file only tests for whether the event sends the expected api request. I don't think any changes need to be made since there are no new events.